### PR TITLE
Postgresql support

### PR DESCRIPTION
**Pull request by davidwatkins73**
Main changes:
 - created properties for booleans: `true|false` (pg) /  `1|0` (mssql/mariadb)
 - constraints must have 'global names

#3599

### CODE REVIEW

**Comment by davidwatkins73** [comment]
Spun up a test instance and ran `...jobs...LoadAll` then tested the ui.  All looks good.  Needs more extensive testing before we officially announce support for this.

**Comment by davidwatkins73** [comment]
Doh, forgot to do searches

File: waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml [hunk 1]
```diff
@@ -2096,7 +2108,8 @@
 
 
     <changeSet author="dwatkins"
-               id="20160629-253-3">
+               id="20160629-253-3"
+               dbms="mssql,maria">
```
Comment (kamransaleem) [inline waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml]: maria or mysql?

Comment (davidwatkins73) [reply waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml]: Good spot. Will update tomorrow

File: waltz-data/src/main/ddl/liquibase/db.changelog-1.7.xml [hunk 2]
```diff
@@ -299,6 +300,20 @@
         </sql>
     </changeSet>
 
+    <changeSet id="20171024-2623-2"
+               author="kamransaleem"
+               dbms="postgresql">
+        <sql>
+            UPDATE data_type
+            SET unknown = TRUE
```
Comment (rovats) [inline waltz-data/src/main/ddl/liquibase/db.changelog-1.7.xml]: Can't we use `unknown = {true.value}` here and just modify the original changeset to include `postresql` as well?

Comment (davidwatkins73) [reply waltz-data/src/main/ddl/liquibase/db.changelog-1.7.xml]: I was getting odd digest failures for some `true.value` substitutions. Think this was one of them. No idea why some don't seem to affect the digest and some do...

File: waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml (hunk 1 above)
Comment (davidwatkins73) [reply waltz-data/src/main/ddl/liquibase/db.changelog-1.0.xml]: Fixed

**Pull request merged**